sirepo/template/raydata.py

```diff
@@ -125,6 +125,10 @@ def stateless_compute_scans(data):
     return _scan_info_result(s)
 
 
+def stateless_compute_get_columns(data):
```

Comment:
The `scans_info` method needs to change to only support getting the fields that the user has selected.

Reply:
07e13b7b7da132d5450c7714b0bcd7a04982308c